src/command-export-html.js

```diff
@@ -163,52 +163,9 @@ export default function(context) {
       hasFixedLayers
     };
 
-    let findHotspotsUnderSubtree_ = (nativeParentGroup, hotspotOverrides) => {
-      let layersWithFlow = common.getAllLayersMatchingPredicate(
-          nativeParentGroup,
-          NSPredicate.predicateWithFormat('flow != nil'));
-      for (let nativeLayer of layersWithFlow) {
-        let layerId = String(nativeLayer.objectID());
-        let nativeFlow = nativeLayer.flow();
-        let frame = nativeLayer.frame();
-        let rectangle = new Rectangle(frame.x(), frame.y(), frame.width(), frame.height());
-        let isFixed = common.isLayerFixedToViewport(nativeLayer);
-        let outermostFixedToViewportParent = isFixed ? nativeLayer : null;
-        let parent = nativeLayer.parentGroup();
-        while (parent && !(parent instanceof MSArtboardGroup || parent instanceof MSSymbolMaster)) {
-          rectangle.offset(parent.frame().x(), parent.frame().y());
-          if (common.isLayerFixedToViewport(parent)) {
-            outermostFixedToViewportParent = parent;
-          }
-          parent = parent.parentGroup();
-        }
-
-        if (outermostFixedToViewportParent) {
-          isFixed = true;
-          // hotspots inside floating/fixed layers that are a fixed distance to the
-          // right or bottom of the screen should be positioned based on viewport size,
-          // not artboard size
-          if (outermostFixedToViewportParent.hasFixedRight()) {
-            rectangle.offset(viewportSize.width - artboardSize.width, 0);
-          }
-          if (outermostFixedToViewportParent.hasFixedBottom()) {
-            rectangle.offset(0, viewportSize.height - artboardSize.height);
-          }
-        }
-
-        let target = String(nativeFlow.destinationArtboardID());
-        if (layerId in hotspotOverrides) {
-          target = hotspotOverrides[layerId];
-        }
+    let dupList = [];
 
-        if (target && nativeFlow.isValidFlowConnection()) {
-          if (target !== String(Flow.BackTarget)) {
-            artboardsToProcess.push(target);
-          }
-
-          artboardData.hotspots.push({rectangle, target, isFixed});
-        }
-      }
+    let findHotspotsUnderSubtree_ = (nativeParentGroup) => {
```

Comment:
seems like this is now just flattening the subtree's symbols / recursively detaching from symbol. may want to rename the function to indicate that's what's happening

Reply:
Sure, of course :)